SecuritySystemUWP/SecuritySystemUWP/Camera/Webcam.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Gpio;
+using Windows.Storage;
+using System.Collections.Generic;
+using Microsoft.Maker.Devices.Media.UsbCamera;
+using Microsoft.Maker.Devices.Gpio.PirSensor;
+
+namespace SecuritySystemUWP
+{
+    public class Webcam : ICamera
+    {
+        private UsbCamera webcam;
+        private PirSensor pirSensor;
+        private bool isCapturing;
```

Comment:
bool read/writes in C# are atomic: https://msdn.microsoft.com/en-us/library/aa691278(VS.71).aspx